<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1401</width>
    <height>818</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>341</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Ja</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nein</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>341</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>An</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Aus</string>
    </property>
   </item>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>511</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_eingabe">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>770</y>
     <width>88</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Eingabe</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>151</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Rollladen unten?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>151</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Licht?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>331</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Bitte gebe Sie die folgenden Daten ein:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>201</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Fenster Ausrichtung in Grad</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>390</y>
     <width>181</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Fensterposition auswählen</string>
   </property>
  </widget>
  <widget class="QLabel" name="Bild">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>521</width>
     <height>341</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1677</width>
     <height>471</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>arbeitsplatz.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDial" name="dial">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>550</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="pageStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>270</number>
   </property>
   <property name="sliderPosition">
    <number>270</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
   <property name="notchesVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>290</y>
     <width>341</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>hell</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>mittel</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>dunkel</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>151</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Möbel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Ausgabe">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>50</y>
     <width>201</width>
     <height>151</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Eingabe betätigen...</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>10</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Eingaben</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>30</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="LuxWert">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>420</y>
     <width>241</width>
     <height>151</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Auswertung...</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>10</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Wetterdaten</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>30</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="WetterDaten">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>50</y>
     <width>281</width>
     <height>141</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Eingabe betätigen...</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>380</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Auswertung</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>380</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Systemdaten</string>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>400</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>400</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="Systemdaten">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>430</y>
     <width>411</width>
     <height>251</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Auswertung...</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
